packages/shared/lib/profile.ts

```diff
@@ -88,18 +88,22 @@ export const saveProfile = (profile: Profile): Profile => {
  *
  * @returns {Profile}
  */
-const buildProfile = (profileName: string, isDeveloperProfile: boolean): Profile => ({
+const buildProfile = (
```

Comment:
This should be a "regular" `function`.

Reply:
to be moved in refactor of profile to lib.